config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -1,5 +1,7 @@
 presubmits:
   kubernetes/kubernetes:
+  # This job is lighweight version of release blocking periodic job 'ci-kubernetes-e2e-gci-gce-scalability'.
+  # Some elements (like access tokens) are removed to speed up testing.
```

Comment:
Thanks!

Let's do the same for release branches, e.g.:
https://github.com/kubernetes/test-infra/blob/9d8ab99aabf76f00432689bcae10836add0f6726/config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

Reply:
Done. I agree that there we do not really need comments for eng onboarding.